assets/js/romo/inline.js

```diff
@@ -40,18 +36,13 @@ RomoInline.prototype.doInit = function() {
   // override as needed
 }
 
-RomoInline.prototype.doInvoke = function() {
-  this.elem.show();
-  this.toggleElem.hide();
-  this.elem.trigger('inline:invoke', [this]);
-}
-
 RomoInline.prototype.doLoadStart = function() {
   this.elem.html('');
   this.elem.trigger('inline:loadStart', [this]);
 }
 
 RomoInline.prototype.doLoadSuccess = function(data) {
+  this.doShow();
```

Comment:
@kellyredding - I don't think this is an issue but previously this was also done before `doLoadError` because we were triggering an invoke with the load error event: https://github.com/redding/romo/pull/54/files#diff-2374be2cb774b569c1acd4ce5e83af7cR74.